### DISCUSSION TITLE
Extend support to watch Github Actions

### DISCUSSION THREAD

**Discussion by jpike88** [deszip/BitBot#54] [Feature requests]
This is a great tool, and it would be great if this could be extended not just to watch Bitrise jobs, but Github Actions too.

Thoughts?

**Comment by deszip** [comment]
Hi @jpike88, agree, this idea also crossed my mind some time ago but building a proper level of abstraction to plug other services appeared to be a non one evening task :) Will think of it more... Thanks for a suggestion.

Will convert this to a discussion, feel free to share other related thoughts here
